### ISSUE TITLE
Problem creating admin user for new install

### ISSUE THREAD

**Issue by wimlmg** [revive-adserver/revive-adserver#548]
The following error message occurs, and can be repated, when trying to do a fresh installation:

****************************
An error occurred while accessing the database

Due to a problem with the database Revive Adserver couldn't retrieve or store data. If this problem is reproducable it might be caused by a bug in Revive Adserver. Please report the following information to the creators of Revive Adserver. Also try to describe the actions that led to this error as clearly as possible.

Version:    Revive Adserver v3.2.0 
PHP/DB: PHP 5.5.24 / MySQL 5.6.23- 
Page: /revive/www/admin/install.php 
Error: Duplicate entry 'default_banner_image_url' for key 'rv_preferences_preference_name' 
Query: INSERT INTO rv_preferences (preference_name , account_type ) VALUES ('default_banner_image_url' , 'TRAFFICKER' )  
$_POST: Array
(
    [_qf__install-config-form] => 
    [token] => 
    [action] => configuration
    [moreFieldsShown] => 0
    [adminName] => xxxxxx
    [adminPassword] => ***
    [adminPassword2] => ***
    [adminEmail] => wg@example.com
    [adminLanguage] => en
    [prefsTimezone] => Europe/Luxembourg
    [webpathAdmin] => www.***/revive/www/admin
    [webpathDelivery] => www.***/revive/www/delivery
    [webpathImages] => www.***/revive/www/images
    [webpathDeliverySSL] => www.***/revive/www/delivery
    [webpathImagesSSL] => www.***/revive/www/images
    [storeWebDir] => /home/***/public_html/revive/www/images
    [save] => Continue »
)
 
$_GET: Empty 
***********************
In short, as someone new to Revive Adserver, I cannot install the software. It is repeatable.

Kind regards,
Wim Geeraets

**Comment by erikgeurts** [comment]
I've removed your e-mail address from your post, but it has been on there for some time. You are most likely going to see an increase in the amount of spam in your mailbox. I also hope that domain in your e-mail address was not the same as that of your ad server, because you've also included your username in your post (which I've also just removed).

The error message you have there, seems to indicate it's attempting to insert the same record twice in the preferences table:
```Error: Duplicate entry 'default_banner_image_url' for key 'rv_preferences_preference_name' 
Query: INSERT INTO rv_preferences (preference_name , account_type ) VALUES ('default_banner_image_url' , 'TRAFFICKER' )```

I've not been able to reproduce this on a new installation of Revive Adserver v3.2.0.

Has anyone in the community seen similar issues?

**Comment by wimlmg** [comment]
Hi Erik,

 

I thought I had removed everything from the post.

I did a search to see whether anyone had seen anything similar, but couldn’t find anything, hence the ticket. 

 

The ticket can be closed now, I just managed to get it working, after completely removing whatever was installed already first.

It appears that this happens when not all permission are set yet, and one continues after having set all required permissions, without cleaning out the database.

 

Kind regards,

Wim Geeraets

 

P.S.: Are you Dutch by any chance?

**Comment by erikgeurts** [comment]
Thanks for your update, I'm glad it was resolved. I will close this issue now.

Yes, I'm very Dutch :)

**Comment by wimlmg** [comment]
Graag gedaan.

 

Ik ben ook Nederlander J.

 

Groetjes,

Wim

 

Wim L.M. Geeraets

IT Consultancy

Blue Maple B.V.

+31 6 50 20 1000
